README.md

```diff
@@ -80,10 +80,9 @@ To learn more about the platform, check out our [documentation](https://github.c
 
   ```sh
   git clone https://github.com/ostis-ai/ostis-web-platform
```

Comment:
This section should replace Natively section

Reply:
Once the component manager has been fully tested and his minimal functionality has been implemented, then the native installation can be removed. Now both options should be available.